src/components/Table/sgds-table.ts

```diff
@@ -38,6 +43,9 @@ export class SgdsTable extends SgdsElement {
    */
   @property({ type: String, reflect: true }) headerPosition: HeaderPosition = "horizontal";
 
+  /** @internal */
+  @state() tableContents: Array<Node> = [];
```

Comment:
```suggestion
  @state() private tableContents: Array<Node> = [];
```

then can remove /** @internal **/